### DISCUSSION TITLE
Urgent: Report of a Phishing URL Hosted on Vercel

### DISCUSSION THREAD

**Discussion by Faizaanahmed** [vercel/community#4986] [Help] [answered]
### Summary

Vercel being used as a Phishing Portal - 

### Steps to Reproduce

Dear Vercel Support Team,

I am writing to urgently bring to your attention a malicious website hosted on your platform, which I believe is being used for phishing activities. The URL in question is https://nvcdf.vercel.app/a2.

Upon visiting this site, it is immediately apparent that it is designed to mimic a legitimate login page, with fields for a username, password, mobile number, and a CAPTCHA-like verification system. This setup is a common tactic used in phishing schemes to illicitly obtain sensitive personal information from unsuspecting individuals.

As a responsible internet user and in the interest of online safety, I strongly urge you to take immediate action against this site. Allowing such a site to remain active not only poses a significant risk to the public but also potentially damages the reputation of Vercel as a hosting service provider.

I understand the challenges involved in monitoring and regulating content on such a large platform. However, in instances like this, where there is a clear and present danger to user security, prompt action is crucial.

I am available to provide any further information you might need to assist in your investigation. I appreciate your prompt attention to this serious matter and look forward to your confirmation of the site’s removal.

Thank you for your cooperation and commitment to maintaining a safe and secure internet environment.

Best regards,

**Comment by Faizaanahmed** [comment]
![Screenshot 2023-11-27 164407](https://github.com/vercel/community/assets/25970667/174e787f-5fda-4cde-b8b9-1e6182157bda)

**Comment by amyegan** [comment, marked as answer]
Thank you for reporting this. Could you also send this report to abuse@vercel.com so a ticket will be generated? I want to be sure it is tracked appropriately in our system. 🙏

**Comment by Faizaanahmed** [comment]
Thank you for your response, I have sent an email. Also, will it be possible to share the contact details of the person who created this so that I can reach out to the authorities.

**Reply by amyegan** [reply]
Thank you for sending that email. I can't publicly share private account information. We will investigate and determine the appropriate course of action to remove safety and security threats.

**Reply by Faizaanahmed** [reply]
Sure, I do suggest you to please take prompt action on this. The app is still up. Time is of the essence here as the longer the app stays, the chance of people losing money increases. I have already taken this up with the authorities, and having additional information will help nab the culprits sooner

**Comment by sreejithag** [comment]
@amyegan  exact same template again on this 

I think Vercel don't care about phishing seriously and helping scammers 

https://srdt8af.vercel.app/gcw

<img width="1256" alt="Screenshot 2023-12-24 at 12 20 59" src="https://github.com/vercel/community/assets/22855663/6bf7b806-32b2-482e-a28e-67bca5d1e1f4">

**Reply by swarnava** [reply]
Please email us at abuse@vercel.com. Unofortunately, We don't monitor abuse report via community, Thank you!

**Reply by wahluf** [reply]
hey @swarnava . i also send email to your team from november at abuse@vercel.com. till now, still no action. 
please, share how to we can boost for quict action. i send you more than 10 link btw.

**Reply by Faizaanahmed** [reply]
I really do suggest that you take done such apps at the earliest or build some system so that your portal isn't used for scams. 
This whole service is asking for a lawsuit
